response_operations_ui/controllers/party_controller.py

```diff
@@ -125,6 +130,48 @@ def search_respondent_by_email(email):
     return response.json()
 
 
+def search_respondents(first_name, last_name, email_address, page=0):
+    if bool(os.environ.get('FAKE_PARTY_DATA', False)):
```

Comment:
Currently, this uses an environment variable to allow switching during session, but this would be replaced in later with test modes that don't affect the wider system.